ui/app/src/components/Launcher/Launcher.tsx

```diff
@@ -406,23 +401,7 @@ export function Launcher(props: LauncherStateProps) {
   const onSiteCardClick = (site: string) => {
     if (!checkMinimized()) {
       setSiteCookie(site, useBaseDomain);
-      fetchUseLegacyPreviewPreference(site, activeEnvironment).subscribe((useLegacy) => {
-        if (!useLegacy && window.location.href.includes(PREVIEW_URL_PATH)) {
-          // If user is in UI next and switching to a site that's viewed in 4.
-          dispatch(batchActions([changeSite(site), closeLauncher()]));
-        } else {
-          // If we're in legacy preview already (i.e. switching from a legacy-preview site to another legacy-preview
-          // site) only the hash will change and the page won't reload or do anything perceivable since legacy isn't
-          // fully integrated with the URL. In these cases, we need to programmatically reload.
-          window.location.href = getSystemLink({
-            systemLinkId: 'preview',
-            authoringBase,
-            useLegacy,
-            site
-          });
-          useLegacy && window.location.reload();
-        }
-      });
+      dispatch(batchActions([changeSite(site), closeLauncher()]));
```

Comment:
What if you aren't on preview? Like in Global or Project tools? I think this breaks that.